parsl/channels/ssh/ssh.py

```diff
@@ -26,7 +26,8 @@ class SSHChannel(Channel, RepresentationMixin):
 
     '''
 
-    def __init__(self, hostname, username=None, password=None, script_dir=None, envs=None, gssapi_auth=False, skip_auth=False, port=22, key_filename=None):
+    def __init__(self, hostname, username=None, password=None, script_dir=None, envs=None,
+                 gssapi_auth=False, skip_auth=False, port=22, key_filename=None, host_keys_file=None):
```

Comment:
host_keys_file is a new user-facing feature so maybe should be mentioned in the PR title.

Reply:
use `filename`, not `file`, for consistency with `key_filename`.

Reply:
Good point. Renamed and added note in the PR description.